inference-engine/src/plaidml_plugin/ops/lstm_cell.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <limits>
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset4.hpp"
+#include "plaidml/op/op.h"
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+edsl::Tensor clip_Tensor(bool should_clip, float clip, const edsl::Tensor& T) {
+    if (should_clip) {
+        return op::clip(T, edsl::Tensor(-clip), edsl::Tensor(clip));
+    } else {
+        return T;
+    }
+}
+
+edsl::Tensor activation_func(std::string func_name, const edsl::Tensor& T) {
+    if (func_name == "relu") {
+        return op::relu(T);
+    } else if (func_name == "sigmoid") {
+        return op::sigmoid(T);
+    } else if (func_name == "tanh") {
+        return edsl::tanh(T);
+    } else {
+        THROW_IE_EXCEPTION << "Unsupported activation function";
+    }
+}
+
+static OpRegistration reg("lstmcell", [](const Context& ctx) {
+    IE_ASSERT(ctx.operands.size() == 6);
+    auto Xt = ctx.operands.at(0);  // input tensor
+    auto Ht_1 = ctx.operands.at(1);  // hidden state tensor
+    auto Ct_1 = ctx.operands.at(2);  // cell state tensor
+    auto W = ctx.operands.at(3);  // weight tensor [4 * hidden_size, input_size]
+    auto R = ctx.operands.at(4);  // recurrence weight tensor [4 * hidden_size, input_size]
+    auto B = ctx.operands.at(5);  // bias tensor [4 * hidden_size]
+
+    auto input_size = Xt.compute_shape().sizes().back();
+    auto* layer = ngraph::as_type<ngraph::opset4::LSTMCell>(ctx.layer);
+    auto hidden_size = layer->get_hidden_size();
+
+    auto activations = layer->get_activations();
+    auto activation_f = activations.at(0);
+    auto activation_i = activations.at(1);
+    auto activation_o = activations.at(2);
+
+    // TODO: activation_alpha and activation_beta are not used
+    auto activation_alpha = layer->get_activations_alpha();
+    auto activation_beta = layer->get_activations_beta();
```

Comment:
My guess is that alpha & beta are reserved names for if more activations are added in the future, and for now they are unused. I would just ignore them altogether.